src/components/List/List.jsx

```diff
@@ -0,0 +1,44 @@
+import React, {useState} from 'react';
+import ListElement from "../ListElement/ListElement";
+
+function List() {
+  const [inputVal, setTodo] = useState("");
+  const [todoList, setTodoList] = useState([]);
+
+  const onButtonClick = (e) => {
+    let newElement = {
+      id: Date.now(),
+      inputVal,
+      status: "done"
+    };
+    e.preventDefault();
+    setTodoList([...todoList, newElement]);
+    console.log(inputVal)
+  };
+
+  return (
+    <>
+      <span className="title">todo list 1.0</span>
+        <form onSubmit={onButtonClick} className="input-group">
```

Comment:
Ideally, you commit code that is proper (in this case formatted). Linter and Prettyfier can help you with that.